ethcore/src/spec/spec.rs

```diff
@@ -98,6 +98,8 @@ pub struct CommonParams {
 	pub block_reward: U256,
 	/// Registrar contract address.
 	pub registrar: Address,
+	/// Apply reward
+	pub apply_reward: bool,
```

Comment:
Yeah, it should be in the `engines::common` module for code reuse. But not part of the `Engine` trait itself!